### ISSUE TITLE
Multiselect for various options

### ISSUE THREAD

**Issue by Joly0** [fmasa/wfrp-master#265]
**Is your feature request related to a problem? Please describe.**
I was trying to re-import the rulebook, but found it very tedious to have to select every single skill/talent/whatever and manually remove every single one of them. 

**Describe the solution you'd like**
There should be an option to multi-select multiple things, so i can bulk-remove them in one go. 
I guess this could also be useful for other things like the trappings, when you want to add multiple trappings to a container in bulk.

Additionally a button "select all" would be useful.

**Describe alternatives you've considered**
None

**Additional context**

**Comment by fmasa** [comment]
Yes. I can imagine this would be useful in several places. It would mean that I would have to come up with different interaction for context menu (currently there are actions such as "Remove" hidden under long press), which is doable but quite a lot of work. Or maybe reusing this context menu and adding some "

> I was trying to re-import the rulebook, but found it very tedious to have to select every single skill/talent/whatever and manually remove every single one of them.

Can you elaborate? When reimporting Rulebook, you can just check "Select All" when importing from rulebook and the items will replace the existing items with the same name rather than be imported as a new item:

![image](https://github.com/fmasa/wfrp-master/assets/5658260/71217dcf-58c5-4f90-922d-f20a9a811fad)

**Comment by Joly0** [comment]
Hey, thanks for you answer. Yes i know the import function replaces items with the same name, but i am trying to export the rulebook, translate it to german manually on my pc and then reimport the translated rulebook, which then doesnt replace the existing entries
